src/assets/scripts/client/constants/canvasConstants.js

```diff
@@ -14,3 +14,21 @@ export const DEFAULT_CANVAS_SIZE = {
     HEIGHT: 480,
     WIDTH: 640
 };
+
+/**
+ * Various options for drawing aircraft on canvas
+ *
+ * @enum AIRCRAFT_DRAW_OPTIONS
+ * @type {Object}
+ * @final
+ */
+export const AIRCRAFT_DRAW_OPTIONS = {
+    /**
+     * Length of PTL lines, in minutes. Standard is 1.
```

Comment:
Wha, you think I don't know that? I almost ended my last response with `Oh, and before you ask... FAA = Federal Aviation Administration && STARS = Standard Terminal Automation Replacement System`. _shots fired!_ 💥